src/server/bin.js

```diff
@@ -25,6 +28,16 @@ async function run () {
   const libp2pMa = argv.libp2pMultiaddr || argv.lm || process.env.LIBP2PMA || '/ip6/::/tcp/5892/ws'
   const addresses = [multiaddr(libp2pMa)]
 
+  let peerInfo
+  if (argv.peerId) {
+    const peerData = fs.readFileSync(argv.peerId)
+    const peerId = await PeerId.createFromJSON(JSON.parse(peerData))
+    peerInfo = await PeerInfo.create(peerId)
+  } else {
+    log('You are using an automatically generated peer. \n')
```

Comment:
Doing 2 logs so it's going to get a newline automatically.
```suggestion
    log('You are using an automatically generated peer.')
```